### PR TITLE
R in Goverment Community Call

### PR DESCRIPTION
**Pull request by yabellini**
Placeholder

### CODE REVIEW

File: content/commcalls/2023-10-31/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: ""
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:28]:
```suggestion
    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
```

File: content/commcalls/2023-10-31/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    img: luisa-andrade.png
+  - name: Karly Harker
+    bio: ""
+    img: karly-harker.jpg
+  - name: Ahmadou Dicko
+    bio: ""
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:34]:
```suggestion
    bio: "I am a statistician and R enthusiast. I believe in the use of data science for social good and have worked with several non-profit and research organizations."
```

File: content/commcalls/2023-10-31/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+speakers:  
+  - name: Luíza Andrade
+    bio: ""
+    img: luisa-andrade.png
+  - name: Karly Harker
+    bio: ""
+    img: karly-harker.jpg
+  - name: Ahmadou Dicko
+    bio: "I am a statistician and R enthusiast. I believe in the use of data science for social good and have worked with several non-profit and research organizations."
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:34]:
```suggestion
    bio: "Ahmadou Dicko is a humanitarian data scientist, currently serving as a Statistics and Data Analysis officer at UNHCR regional bureau for West and Central Africa. He is a fervent advocate for reproducibility in the humanitarian sector and has built several R packages and tools to streamline data processes. With M.Sc. in Statistics and a Ph.D. in Climate Change Economics, Ahmadou combines his profound knowledge of economics and data science to drive transformative change in the humanitarian sector. He is the founder and co-organizer of the Dakar R User Group and a recognized leader in promoting data literacy and community building around humanitarian data science."
```

File: content/author/ahmadou-dicko/_index.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+name: Ahmadou Dicko
+link: https://www.ahmadoudicko.com/about/
+linkedin: https://www.linkedin.com/in/ahmadoudicko 
+mastodon: https://fosstodon.org/@dickoa
+github: dickoa
```
Comment (yabellini) [inline content/author/ahmadou-dicko/_index.md:6]:
```suggestion
github: dickoa
img: img/community/ahmadou-dicko.jpg
```

File: content/commcalls/2023-10-31/index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    img: luisa-andrade.png
+  - name: Karly Harker
+    bio: ""
+    img: karly-harker.jpg
+  - name: Ahmadou Dicko
+    bio: "Ahmadou Dicko is a humanitarian data scientist, currently serving as a Statistics and Data Analysis officer at UNHCR regional bureau for West and Central Africa. He is a fervent advocate for reproducibility in the humanitarian sector and has built several R packages and tools to streamline data processes. With M.Sc. in Statistics and a Ph.D. in Climate Change Economics, Ahmadou combines his profound knowledge of economics and data science to drive transformative change in the humanitarian sector. He is the founder and co-organizer of the Dakar R User Group and a recognized leader in promoting data literacy and community building around humanitarian data science."
+    img: ahmadou-dicko.jpg
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:35]:
```suggestion
    img: img/community/ahmadou-dicko.jpg
```

File: content/commcalls/2023-10-31/index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
+    img: luisa-andrade.png
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:29]:
```suggestion
    img: img/community/luiza-andrade.jpg
```

File: content/commcalls/2023-10-31/index.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  - name: Karly Harker
+    bio: ""
+    img: karly-harker.jpg
+  - name: Ahmadou Dicko
+    bio: "Ahmadou Dicko is a humanitarian data scientist, currently serving as a Statistics and Data Analysis officer at UNHCR regional bureau for West and Central Africa. He is a fervent advocate for reproducibility in the humanitarian sector and has built several R packages and tools to streamline data processes. With M.Sc. in Statistics and a Ph.D. in Climate Change Economics, Ahmadou combines his profound knowledge of economics and data science to drive transformative change in the humanitarian sector. He is the founder and co-organizer of the Dakar R User Group and a recognized leader in promoting data literacy and community building around humanitarian data science."
+    img: ahmadou-dicko.jpg
+  - name: Pablo Tiscornia
+    bio: ""
+    img: pablo-tiscornia.jpg
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:38]:
```suggestion
    img: img/community/pablo-tiscornia.jpeg
```

File: content/commcalls/2023-10-31/index.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+- HTML
+- Calendar 
+# vimeo:
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
+    img: img/community/luiza-andrade.jpg
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:29]:
```suggestion
    img: /img/community/luiza-andrade.jpg
```

File: content/commcalls/2023-10-31/index.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  - name: Karly Harker
+    bio: ""
+    img: karly-harker.jpg
+  - name: Ahmadou Dicko
+    bio: "Ahmadou Dicko is a humanitarian data scientist, currently serving as a Statistics and Data Analysis officer at UNHCR regional bureau for West and Central Africa. He is a fervent advocate for reproducibility in the humanitarian sector and has built several R packages and tools to streamline data processes. With M.Sc. in Statistics and a Ph.D. in Climate Change Economics, Ahmadou combines his profound knowledge of economics and data science to drive transformative change in the humanitarian sector. He is the founder and co-organizer of the Dakar R User Group and a recognized leader in promoting data literacy and community building around humanitarian data science."
+    img: img/community/ahmadou-dicko.jpg
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:35]:
```suggestion
    img: /img/community/ahmadou-dicko.jpg
```

File: content/commcalls/2023-10-31/index.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+  - name: Pablo Tiscornia
+    bio: ""
+    img: img/community/pablo-tiscornia.jpeg
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:38]:
```suggestion
    img: /img/community/pablo-tiscornia.jpeg
```

File: content/commcalls/2023-10-31/index.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
+    img: /img/community/luiza-andrade.jpg
+  - name: Karly Harker
+    bio: ""
+    img: karly-harker.jpg
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:32]:
```suggestion
    img: /img/community/karly-harker.jpeg
```

File: content/commcalls/2023-10-31/index.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+# vimeo:
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
+    img: /img/community/luiza-andrade.jpg
+  - name: Karly Harker
+    bio: ""
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:31]:
```suggestion
    bio: "Environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"
```

**Review by yabellini — COMMENTED** [review]
Adding Karly's info. Using LinkedIn data.

**Review by pablotis — CHANGES_REQUESTED** [review]
Please find below the bio for Pablo Tiscornia:

_Pablo is a graduate in Sociology from the University of Buenos Aires, Argentina. Currently, he works at C40 Cities as a Data Scientist, with a focus on improving data workflows related to Climate Change. He also serves as an instructor for Data Processing with R for Social Sciences at several National Universities in Argentina. 
He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires._

File: content/commcalls/2023-10-31/index.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+    img: /img/community/luiza-andrade.jpg
+  - name: Karly Harker
+    bio: "Environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"
+    img: /img/community/karly-harker.jpeg
+  - name: Ahmadou Dicko
+    bio: "Ahmadou Dicko is a humanitarian data scientist, currently serving as a Statistics and Data Analysis officer at UNHCR regional bureau for West and Central Africa. He is a fervent advocate for reproducibility in the humanitarian sector and has built several R packages and tools to streamline data processes. With M.Sc. in Statistics and a Ph.D. in Climate Change Economics, Ahmadou combines his profound knowledge of economics and data science to drive transformative change in the humanitarian sector. He is the founder and co-organizer of the Dakar R User Group and a recognized leader in promoting data literacy and community building around humanitarian data science."
+    img: /img/community/ahmadou-dicko.jpg
+  - name: Pablo Tiscornia
+    bio: ""
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:37]:
```suggestion
    bio: "Pablo is a graduate in Sociology from the University of Buenos Aires, Argentina. Currently, he works at C40 Cities as a Data Scientist, with a focus on improving data workflows related to Climate Change. He also serves as an instructor for Data Processing with R for Social Sciences at several National Universities in Argentina.
He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires."
```

File: content/commcalls/2023-10-31/index.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires."
+    img: /img/community/pablo-tiscornia.jpeg
+tags:
+  - community call
+  - events
+  - goverment
+deets: |
+    Meeting ID:  
+    Passcode: 
+zoomurl:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:47]:
```suggestion
zoomurl: https://numfocus-org.zoom.us/j/83100290068?pwd=c4bloSdA1zNaQ7IzJD83BHD3I54zab.1
```

File: content/commcalls/2023-10-31/index.md [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+  - name: Pablo Tiscornia
+    bio: "Pablo is a graduate in Sociology from the University of Buenos Aires, Argentina. Currently, he works at C40 Cities as a Data Scientist, with a focus on improving data workflows related to Climate Change. He also serves as an instructor for Data Processing with R for Social Sciences at several National Universities in Argentina.
+He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires."
+    img: /img/community/pablo-tiscornia.jpeg
+tags:
+  - community call
+  - events
+  - goverment
+deets: |
+    Meeting ID:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:45]:
```suggestion
    Meeting ID:  831 0029 0068 
```

File: content/commcalls/2023-10-31/index.md [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+    bio: "Pablo is a graduate in Sociology from the University of Buenos Aires, Argentina. Currently, he works at C40 Cities as a Data Scientist, with a focus on improving data workflows related to Climate Change. He also serves as an instructor for Data Processing with R for Social Sciences at several National Universities in Argentina.
+He has made significant contributions to the development of R as a co-author of various packages used by Argentina's leading official statistical offices (INDEC, MINTURyDEP, and DGEyC). He has also been a co-organizer of the R community in Buenos Aires."
+    img: /img/community/pablo-tiscornia.jpeg
+tags:
+  - community call
+  - events
+  - goverment
+deets: |
+    Meeting ID:  
+    Passcode:
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:46]:
```suggestion
    Passcode: 042189
```

File: content/commcalls/2023-10-31/index.md [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+author:
+  - Luíza Andrade
+  - Karly Harker
+  - Ahmadou Dicko
+  - Pablo Tiscornia
+organizer: Yanina Bellini Saibene
+location: Community call (teleconference)
+country: "\U0001F310"
+attendees: All are welcome
+slug: oct2023-goverment
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:17]:
```suggestion
slug: oct2023-government
```

File: content/commcalls/2023-10-31/index.md [hunk 18]
```diff
@@ -0,0 +1,52 @@
...
+  - community call
+  - events
+  - goverment
+deets: |
+    Meeting ID:  831 0029 0068 
+    Passcode: 042189
+zoomurl: https://numfocus-org.zoom.us/j/83100290068?pwd=c4bloSdA1zNaQ7IzJD83BHD3I54zab.1
+---
+
+In this community call, our panelist will share their experiences and examples of projects with R at different levels of government and in different countries.  We invite you to learn about the challenges and lessons learned from our panelists and attendees in their efforts to make their government data, processes, and analysis more open and reproducible.
```
Comment (maelle) [inline content/commcalls/2023-10-31/index.md:50]:
```suggestion
In this community call, our panelists will share their experiences and examples of projects with R at different levels of government and in different countries.  
We invite you to learn about the challenges and lessons learned from our panelists and attendees in their efforts to make their government data, processes, and analyses more open and reproducible.    
```

File: content/commcalls/2023-10-31/index.md [hunk 19]
```diff
@@ -0,0 +1,52 @@
...
+# vimeo:
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
+    img: /img/community/luiza-andrade.jpg
+  - name: Karly Harker
+    bio: "Environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"
```
Comment (maelle) [inline content/commcalls/2023-10-31/index.md:31]: could this one be sentences too?

File: content/author/luiza-andrade/_index.md [hunk 20]
```diff
@@ -0,0 +1,4 @@
+---
+name: Luiza Andrade
```
Comment (maelle) [inline content/author/luiza-andrade/_index.md:2]: no link at all?

**Review by maelle — COMMENTED** [review]
Thank you!

File: content/commcalls/2023-10-31/index.md (hunk 19 above)
Comment (yabellini) [reply content/commcalls/2023-10-31/index.md:31]: Unfortunately I only have that information.

File: content/author/karly-harker/_index.md [hunk 21]
```diff
@@ -0,0 +1,5 @@
+---
+name: Karly Harker
+linkedin: https://www.linkedin.com/in/karlyharker/
```
Comment (yabellini) [inline content/author/karly-harker/_index.md:3]:
```suggestion
linkedin: https://www.linkedin.com/in/karlyharker/
github: KarHarker
```

File: content/commcalls/2023-10-31/index.md (hunk 19 above)
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:31]:
```suggestion
    bio: "Karly Harker is an environmental Data Science Specialist at British Columbia Ministry of Environment and Climate Change"
```

File: content/author/luiza-andrade/_index.md (hunk 20 above)
Comment (yabellini) [inline content/author/luiza-andrade/_index.md:2]:
```suggestion
name: Luiza Andrade
link: https://bfi.uchicago.edu/scholar/luiza-andrade/
```

File: content/commcalls/2023-10-31/index.md [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+# vimeo:
+# - id: 
+resources:
+
+speakers:  
+  - name: Luíza Andrade
+    bio: "Luiza Andrade is the Data Analytics Lead at the University of Chicago's Development Innovation Lab. Her work focuses on incorporating non-traditional data sources into development research, promoting transparency and reproducibility in social sciences, and developing software tools to simplify research data work. Prior to joining DIL, she was a Junior Data Scientist at the World Bank’s Development Impact Evaluation department. Luiza is a Brazilian national and holds a BA and an MSc in economics from the University of Sao Paulo."
+    img: /img/community/luiza-andrade.jpg
+  - name: Karly Harker
+    bio: "Karly Harker is an environmental Data Science Specialist at the British Columbia Ministry of Environment and Climate Change. Karly is an Environment and Resource Studies and Biology graduate from the University of Waterloo.
```
Comment (yabellini) [inline content/commcalls/2023-10-31/index.md:31]:
```suggestion
    bio: "Karly Harker is an environmental Data Science Specialist at the British Columbia Ministry of Environment and Climate Change. Karly is an Environment and Resource Studies and Biology graduate from the University of Waterloo."
```

**Pull request merged**